src/UniGetUI.Core.Settings/SettingsEngine.cs

```diff
@@ -110,10 +110,10 @@ public static bool AreUpdatesNotificationsDisabled()
             return AreNotificationsDisabled() || Get("DisableUpdatesNotifications");
         }
 
-        public static bool AreShortcutsNotificationsDisabled()
+        /*public static bool AreShortcutsNotificationsDisabled()
```

Comment:
How come you don't want the user to be able to disable shortcut notifications specifically?

Reply:
The whole point of users enabling icon removal is to get notified and know when new icons are detected, so they can set them for removal.

Reply:
Furthermore, it causes confusion, because you can enable said setting and have disabled the "Delete new shortcut options", and then shortcuts won't be enabled